src/components/Data/Documents/Page.vue

```diff
@@ -661,6 +657,35 @@ export default {
     }
   }
 }
+
+function dateFromTimestamp(value) {
+  let timestamp
+
+  if (typeof value === 'string') {
+    timestamp = parseInt(value, 10)
+
+    if (isNaN(timestamp)) {
+      return null
```

Comment:
What about displaying something instead of null?
```suggestion
      return `Invalid Date value (${value})`
```